lib/src/services/files.dart

```diff
@@ -13,6 +13,10 @@ import 'package:rover_dashboard/src/widgets/navigation/footer.dart';
 
 import "service.dart";
 
+extension on DateTime{
+  String timeStamp() => "$year-$month-$day-$hour-$minute"; 
```

Comment:
```suggestion
  String get timeStamp => "$year-$month-$day-$hour-$minute"; 
```
This makes it so that you can do `.timestamp` instead of `.timestamp()`.

Also -- what about AM/PM? Unrealistic that someone would get on the dashboard _exactly_ 12 hours later, but can we make this 24hr time for readability purposes anyway?